data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,107 @@ mission "Unfettered Jump Drive 3"
+				goto meet
+			label grab
+			`	You manage to grab the weapon, just in time to deflect an incoming attack from the same Unfettered. You duel with them for what feels like an eternity, and you are astonished by the fact you managed to survive that long. You of course do not manage to strike a single successful blow, but at least you are still standing, although in sweats.`
+			`	Then, suddenly, you can feel them getting serious, as if the game was over. They strike a masterful blow that disarms you with ease before smiling and lowering their weapon. "It looks like you know what to do with a blade! I am Desneeparu, and it is pleasure to meet you, captain <last>!" She says, before laughing at your surprised face. You realize she must have been toying with you, but nevertheless she seems satisfied by your performance`
+			label meet
+			action
+				log "minor people" "Desneeparu" `An Unfettered warrior that knows how to handle a blade. She fought you to test your resolve before you were given access to their weapons. You suspect she was holding back or you would never have stood a chance.`
+			`	Now that you have successfully passed the test, you are then taken to an underground zone, passing through what looks like very basic weapon testing. Many Hai are there, experimenting with small modifications, but never performing any extreme tests. Seeing your interest, the engineer explains. "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually shoots before Unfettered captains can get their hands on it." You now notice a few captains looking at the tests, and one even requests a weapon to be taken to his ship, apparently satisfied with its performances. You keep on walking until you enter a proper outfitter section of the facility.`
+			`	"Welcome to the special section of the outfitter, Captain! It is only for our most distinguished guests. Now that you have sold us multiple jump drives, I can tell you that we offer something much more valuable than money. For now, I can only offer you the mighty Tripulse Shredder, but eventually we will give you access to all kinds of weapons. 2 Tripulse Cannons for one jump drive and our sympathy is the deal I propose," they say, presenting the weapons to you. Now that you can take a closer look, you notice these weapons seem fairly similar to the rest of Hai weaponry, but enhanced one way or another. There even are some weapons that appear to be ion-based, but one is smaller than an Ion Cannon whilst another one looks like a turreted version of it.`
```

Comment:
Is it a Shredder or a Cannon?

Reply:
These are the ion weapons here not the shredder